tests/unit/utils/test_utils.py

```diff
@@ -818,3 +818,26 @@ def test_libgl_workaround(monkeypatch, skip):
     if skip:
         monkeypatch.setenv('QUTE_SKIP_LIBGL_WORKAROUND', '1')
     utils.libgl_workaround()  # Just make sure it doesn't crash.
+
+
+@pytest.mark.parametrize('durations, out', [
+    ("-1s", -1),  # No sense to wait for negative seconds
+    ("-1", -1),
+    ("34ss", -1),
+    ("0", 0),
+    ("0s", 0),
+    ("59s", 59000),
+    ("60", 60000),
+    ("60.4s", -1),  # Only accept integer values
```

Comment:
I don't know if anybody cares for this functionality, but supporting it is pretty simple, so maybe it should be done? (This seems nice for `1.5h` instead of `1h30m`)

Reply:
If it's as straightforward as we suspect (replacing `int` -> `float` and also looking for `\.` in the regex), I'm all for it.